submission/Motivator/script.js

```diff
@@ -0,0 +1,86 @@
+// Global variables
```

Comment:
<img width="313" alt="image" src="https://github.com/user-attachments/assets/abb0493a-f797-47a7-bfde-e5042eef3051">
A few suggestions/feedback
1. "Remove" doesn't really check off a task so it can be confusing at first
2. The progress bar doesn't work. It'll be better to have another button close to the "Remove" button for actually checking off a task and moving to the next